### PR TITLE
Compile new master.

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@aashish24 @scottwittenburg Please review.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-{% set version = "8.1.0" %}
+{% set sha = "3056e46" %}
+{% set version = "8.1.0." ~ sha %}
```
Comment (aashish24) [inline recipe/meta.yaml:2]: @danlipsa what does this do?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -53,9 +53,9 @@ requirements:
     - python
     - future  # used in the generated python wrappers
     # VTK Third Party dependencies
-    - zlib 1.2.11
-    - freetype 2.8.1
-    - hdf5 1.10.1
+    - zlib 1.2.11.*
+    - freetype 2.8.1.*
+    - hdf5 1.10.1.*
```
Comment (aashish24) [inline recipe/meta.yaml:58]: since we took out the   -DVTK_USE_SYSTEM_HDF5:BOOL=ON  above, do we still need hdf5?

File: recipe/meta.yaml (hunk 1 above)
Comment (danlipsa) [reply recipe/meta.yaml:2]: Appends the sha to the file name so that you know which VTK you have.

File: recipe/meta.yaml (hunk 2 above)
Comment (danlipsa) [reply recipe/meta.yaml:58]: I think we do - netcdf seems to be always on and that needs hdf5. I think there is a problem with system htf5 with the new VTK - I get an undefined symbol there. I think there has been work on updating third parties which might cause it.

Comment (aashish24) [reply recipe/meta.yaml:58]: I am confused, if you took out the VTK_USE_SYSTEM_HDF5, VTK will use its own mangled HDF5 or did I miss something?

Comment (danlipsa) [reply recipe/meta.yaml:58]: Yes, that is right. The mangled hdf5 works fine.

Comment (aashish24) [reply recipe/meta.yaml:58]: ah, okay. thanks.

**Review by aashish24 — APPROVED** [review]

**Pull request merged**